packages/sanity/src/core/studio/screens/RequestAccessScreen.tsx

```diff
@@ -0,0 +1,200 @@
+/* eslint-disable i18next/no-literal-string,@sanity/i18n/no-attribute-string-literals */
+import {Box, Button, Card, Dialog, Flex, Stack, Text, TextInput, useToast} from '@sanity/ui'
+import {useCallback, useEffect, useState} from 'react'
+import {
+  type CurrentUser,
+  getProviderTitle,
+  LoadingBlock,
+  type SanityClient,
+  useActiveWorkspace,
+} from 'sanity'
+
+import {NotAuthenticatedScreen} from './NotAuthenticatedScreen'
+
+interface AccessRequest {
```

Comment:
Not sure if this is where this type should go? I believe this will be the only screen to use it but would prefer to follow standard practice for codebase if applicable

Reply:
I think this seems sensible